pkg/datapath/loader/types/types.go

```diff
@@ -5,25 +5,41 @@ package types
 
 import (
 	"context"
+	"slices"
 
 	"github.com/vishvananda/netlink"
 
 	"github.com/cilium/cilium/pkg/datapath/loader/metrics"
+	"github.com/cilium/cilium/pkg/datapath/tables"
 	"github.com/cilium/cilium/pkg/datapath/tunnel"
 	"github.com/cilium/cilium/pkg/datapath/types"
 )
 
+// LoaderContext are the external inputs to the loader resolved by the orchestrator.
+type LoaderContext struct {
```

Comment:
Not quite sure about the name. I'm sure it would be confused with `context.Context`. Calling it `State` isn't quite right either, as it really is the "context" in which the loader operation happens.